sdk/core/System.BinaryData/src/TaskExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace System
+{
+    internal static class TaskExtensions
```

Comment:
To my knowledge, we don't have anything like this in dotnet/runtime, which means it'd be out of place in this package once it moves.

I recommend just removing it.